src/slack_interaction_handler.ts

```diff
@@ -434,7 +436,7 @@ export async function receiveResetDemo({
     );
   }
 
-  await SlackApiUtil.renderModal(payload.trigger_id, slackView);
+  await SlackApiUtil.updateModal(viewId, slackView);
```

Comment:
Previously, if an error was thrown above (Interaction received for voter who has redisData but not userInfo), then nothing happens, which isn't ideal (preferably we'd show a modal). After this change, the Loading... modal would appear but never be updated. Can we add an updateModal call right before throwing an error? And perhaps a "Close" to the Loading... modal just in case?

Reply:
Slack automatically adds a close button if you don't specify one, so a user wouldn't be stuck. But updating the modal to say "whoops, there was an error" would be the user-friendly thing to do.